README.md

```diff
@@ -1,15 +1,15 @@
-# The Ultimate Js Guide
+# The Modern JavaScript Guide
 
 ## About
-The purpose of this guide, is to orientate developers about what topics are important to Javascript development and decide what would be their next step to make in their pursuit for mastering the most popular programming language.
+The purpose of this guide, is to show JavaScript developers what topics are important and decide what would be their next step to make in their pursuit for mastering the most popular programming language.

```

Comment:
To me, seems too much imperative to say that we will "show" the path and that we  "know" what is important. But,  we can achieve a more friendly tone by saying something like:

"[..] is to show JavasScript developers what topics we think, at Concrete Solutions, are important and help them decide what could be their next step [..]"

Even if there are times where we must be more strict about what we know , in order to create empathy, we should always try to give the impression that we are mentoring and not instructing people, because that is indeed what we are looking for.

@AlexisNo what do you think? Any suggestions about this intro?